File: estudiante.py
class Estudiante:
    def __init__ (self, nombre, edad , promedio):
        self.edad = edad
        self.nombre = nombre
        self.promedio = promedio
    
    def mostrarInfo(self):
        print(f"Nombre: {self.nombre} , Edad: {self.edad} , Promedio: {self.promedio}")

File: estudiante.java
public class estudiante {
    
    private String nombre;
    private double promedio;
    private int edad;

    public estudiante(String nombre, double promedio, int edad){
        this.edad=edad;
        this.nombre= nombre; 
        this.promedio=promedio;

    }



    public void mostrarInfor(){
        System.out.println("Nombre: " + getNombre() + "Edad: "  + getEdad() + "Promedio: " + getPromedio());
    }


    public String getNombre(){
        return nombre;
    }

    public int getEdad(){
        return edad;
    }
    

    public double getPromedio(){
        return promedio;
    }

    public void setPromedio (double promedio){
        this.promedio = promedio;
    }


}
